model/supervised_finetuning/utils.py

```diff
@@ -37,7 +37,7 @@ class TokenizerConfig(NamedTuple):
 
 def match_tokenizer_name(model_name: str) -> TokenizerConfig:
     """Match a partial model name to a tokenizer configuration"""
-    tokenizer_config_matches = [config for name, config in TOKENIZER_CONFIGS.items() if name in model_name]
+    tokenizer_config_matches = [config for name, config in TOKENIZER_CONFIGS.items() if model_name in name]
```

Comment:
It was already correct, no? We call `match_tokenizer_name` based on the name of the model not the keys of TOKENIZER_CONFIGS

Reply:
Hey @sanagno, yes you're right sorry 😂. I've just updated it back and added some docstrings / tests to make it clearer.